arm/.global/global.module.tests.ps1

```diff
@@ -787,6 +787,36 @@ Describe 'Deployment template tests' -Tag Template {
             $incorrectParameters | Should -BeNullOrEmpty
         }
 
+        It "[<moduleFolderName>] Conditional parameters' description should contain 'Required if' followed by the condition making the parameter required." -TestCases $deploymentFolderTestCases {
+            param(
+                $moduleFolderName,
+                $templateContent
+            )
+
+            if (-not $templateContent.parameters) {
+                # Skip test
+                $true | Should -Be $true
```

Comment:
To skip the test (i.e. Pester will flag it as such), remove this dummy test.
```suggestion
```

Reply:
That is being used in other tests as well. Is there a particular reason why this should be removed here? Or should we remove it from the other tests as well?

Reply:
It should only be there if there is no other way to 'pass' a valid test. I agree that, if it is used incorrectly in other places, we should clean it up. But I hope my rationale makes sense. I'd personally rather have Pester tell me a certain tests didn't run because it's not applicable, rather than 'pretending' it passed.